frontend/src/services/createPostService.js

```diff
@@ -4,15 +4,28 @@ export default async (title, body) => {
   const parameters = {
     title,
     body,
-    sender_created_at: new Date().toISOString(),
   };
 
   const requestBody = JSON.stringify(parameters);
 
-  const response = await fetch('/api/post/create', { method: 'POST', body: requestBody });
+  const response = await fetch('/posts', {
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/json',
+    },
+    method: 'POST',
+    body: requestBody,
+  });
 
   if (!response.ok) {
     const { message } = await response.json();
+    if (message.includes('is required')) {
+      throw new SubmitPostError('Please ensure all fields are filled in');
+    }
     throw new SubmitPostError(message);
   }
+
+  if (response.ok) {
+    window.location.reload();
+  }
```

Comment:
This is the culprit of the failing tests. Do we need this? We will be navigating to the post detail page upon success anyway. Also, we should be using the Router navigation components instead. See https://reacttraining.com/react-router/web/guides/primary-components/navigation-or-route-changers

Reply:
Or maybe not Router navigation, but at least a new `fetch` for the posts?

Reply:
(Context: Reloading the page will mess up the tests).

Reply:
ok I will remove it for now